CHANGELOG.md

```diff
@@ -61,6 +61,9 @@
 * Skip `__wasm_call_ctors` which we don't want to interpret.
   [#4562](https://github.com/wasm-bindgen/wasm-bindgen/pull/4562)
 
+* Fix infinite recursion caused by `BindgenedStruct` derive.
```

Comment:
Instead of internals can we just mention macro hygiene here?
(can also fix this myself when I get home)

Reply:
```suggestion
* Fix infinite recursion caused by the lack of proc-macro hygiene.
```